song-docs/source/server.rst

```diff
@@ -0,0 +1,229 @@
+.. _server_ref:
+
+=======================================
+Deploying a SONG Server in Production
+=======================================
+
+The following section describes how to install, configure and run the SONG server in production.
+
+
+.. _server_prereq:
+
+Prerequisites
+==================
+
+The following software dependencies are required in order to run the server:
+
+- Bash Shell
+- Java 8 or higher
+- Postgres database
+
+.. note::
+    Only a postgres database can be used, since postgres-specific features are used in the SONG server
+
+.. _server_official_releases:
+
+Official Releases
+==================
+
+
+Official SONG releases can be found `here <https://github.com/overture-stack/SONG/releases>`_. The releases follow the `symantic versioning specification <https://semver.org/>`_ and contain notes with a description of the bug fixes, new features or enhancements and breaking changes, as well as links to downloads and change logs. All official song releases are tagged in the format ``$COMPONENT-$VERSION``, where the ``$COMPONENT`` portion follows the regex ``^[a-z-]+$`` and the ``$VERSION`` component follows ``^\d+\.\d+\.\d+$`` . For the SONG server, the tag format has the regex: ``^song-\d+\.\d+\.\d+$``. For example ``song-1.0.0``.
+
+Alternatively, official releases can be found via command line as follows:
```

Comment:
Although I definitely find this useful/interesting, it seems slightly out of scope of the documentation. In a FAQ maybe ?